### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by himadri2110**
Landing page for e-commerce store

Pages:
- Home page
- Products (dummy)
- Cart (dummy)
- Wishlist (dummy)

Components:
- Navbar
- Footer

Assets: 
- Images

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,19 @@
...
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Routes>
```
Comment (poornima-alugubelly) [inline src/App.js]: Can we make a separate route folder and component it is best to have the App file be as clean as possible

**Review by poornima-alugubelly — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (himadri2110) [reply src/App.js]: Yes, I'll do that. Thank you!

**Review by manojasarsa — APPROVED** [review]
The code looks clean and well structured.
Apart from what @poornima-alugubelly suggested, everything looks good!

**Comment by himadri2110** [comment]
> The code looks clean and well structured. Apart from what @poornima-alugubelly suggested, everything looks good!

Yes, I already implemented it, thank you!

**Pull request merged**